service/src/integrationTest/kotlin/fi/espoo/evaka/reservations/MissingHolidayReservationsRemindersTest.kt

```diff
@@ -58,6 +62,8 @@ class MissingHolidayReservationsRemindersTest : FullApplicationTest(resetDbBefor
     private lateinit var child: ChildId
     private lateinit var daycareId: DaycareId
     private lateinit var areaId: AreaId
+    private val employeeId = EmployeeId(UUID.randomUUID())
```

Comment:
Might be better to have here `private val employee = DevEmployee()` and then on line 187 `tx.insert(employee)`. Usually there's no need to call `UUID.randomUUID()` manually when using the latest conventions for test data setup.